tests/test_learning_path.py

```diff
@@ -0,0 +1,66 @@
+from collections import OrderedDict
+import domain.tutoringModel.generate_learning_path as LP
+import pytest
+
+generate_learning_path = LP.GenerateLearningPath()
+
+
+def test_get_learning_path_correct_result():
+    input_learning_style = {"AKT": 5, "INT": 9, "VIS": 9, "GLO": 9}
+    # ZF-ÜB-#AN-#SE-RQ-AB-#ZL-#BE-FO
+    learning_path = {
+        'ZF': 99, 'ÜB': 14, 'SE': 5, 'AN': 5,
+        'RQ': 4, 'AB': 0, 'ZL': -5, 'BE': -5, 'FO': -13}
+    assert generate_learning_path.get_learning_path(
+        input_learning_style) == learning_path
+
+
+def test_get_learning_path_correct_result_2():
+    input_learning_style = {"REF": 1, "SNS": 7, "VIS": 5, "SEG": 5}
+    # result of Learning Path : AN-BE-AB-#SE-#ÜB-RQ-ZF-FO-ZL
```

Comment:
As many comments as necessary, as few as possible

Reply:
I agree, I ll remove this